stablehlo/dialect/StablehloOps.td

```diff
@@ -194,7 +199,7 @@ class StableHLO_UnaryElementwiseOp<string mnemonic, list<Trait> traits,
 
 // Abs supports complex to real, so element type is not guaranteed to match.
 def StableHLO_AbsOp: StableHLO_UnaryElementwiseOp<"abs",
-    [Pure, DeclareOpInterfaceMethods<InferTypeOpInterface>],
+    [DeclareOpInterfaceMethods<InferTypeOpInterface>],
```

Comment:
To clarify - `Pure` can be deleted here and after because the memory effects / speculatability is now captured in the parent `StableHLO_UnaryElementwiseOp` class?

Reply:
That's right! `Pure` is `def Pure : TraitList<[AlwaysSpeculatable, NoMemoryEffect]>;` -- the parent class now specifies `HLO_UnaryElementwiseSpeculatable` and `NoMemoryEffect`, so `Pure` is no longer needed.